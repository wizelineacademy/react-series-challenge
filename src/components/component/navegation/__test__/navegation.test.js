import React from 'react';
import { shallow } from 'enzyme';

import toJson from 'enzyme-to-json';


import Navegation from "./../index";

describe('<Navegation> ', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Navegation />
    );
  });

  it('create a Snapshot ', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});