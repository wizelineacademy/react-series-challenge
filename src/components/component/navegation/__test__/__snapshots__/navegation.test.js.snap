// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navegation>  create a Snapshot  1`] = `
<header>
  <ul>
    <li>
      <navegation__StyledNavLink
        activeClassName="active"
        to="/home"
      >
        HOME
      </navegation__StyledNavLink>
    </li>
    <li>
      <navegation__StyledNavLink
        activeClassName="active"
        to="/keeps"
      >
        Favoritos
      </navegation__StyledNavLink>
    </li>
  </ul>
</header>
`;
